File: Rocket Game/Player.cpp
# include "A_M_Game.h"

Player::Player() = default;

Player::Player(string N, char S){
    Name = std::move(N) ;
    Side = S ;
}

void Player::Information() {
    cout << "The Name -> " << Name <<
    " and his Side -> " << Side << "\n" ;
}

string Player::Get_Name() {
    return Name ;
}

char Player::Get_Side() const {
    return Side;
}

void Player::Set_Name(string n) {
    Name = std::move(n);
}

void Player::Get_Move(int & row, int & col) {
    cout << "Player " << Name << " Enter the coordinate of the rocket\n";
    cin >> row >> col ;
}

File: XO Game/Player.java
import java.util.Scanner;
public class Player {
    private final String Name ;
    private final char Symbol ;
    public Player(String N, char S){
        Symbol = S ;
        Name = N ;
    }
    public String getName(){
        return Name ;
    }
    public char getSymbol(){
        return Symbol ;
    }
    public Move get_move(){
        Scanner in = new Scanner(System.in);
        System.out.print("Please Enter the coordinate of the your play\n==>") ;
        return new Move(in.nextInt(), in.nextInt());
    }
}